### PR TITLE
Fixes #36058 - Drop the socket facts chart

### PR DESCRIPTION
**Pull request by girijaasoni**
Socket Facts chart wasn't rendered completely and there were feedbacks to consider dropping it. We added a community post to get more feedback on if the feature is needed
https://community.theforeman.org/t/socket-chart-feature-is-it-needed/33120
Feedback suggested to drop it. This PR is in reference for the same. Screenshots after dropping it is attached: 
Screenshot from 2023-04-28 12-24-08

### CODE REVIEW

**Review by Ron-Lavi — DISMISSED** [review]
Thanks @girijaasoni !
Looks great, tested and I see no issues with removing that action column,
also found no reference in the foreman org to the FactChart component,
but in the katello org, I found this funny [reference](https://github.com/Katello/katello/blob/master/webpack/components/extensions/about/index.js#L6) which I will rename in a different PR.

ACK from me 👍🏼

File: app/views/fact_values/index.html.erb [hunk 1]
```diff
@@ -17,7 +17,6 @@ end
       <th><%= sort :value, :as => s_("FactValue|Value") %></th>
       <th class="fact-origin"><%= sort :origin, :as => s_("FactValue|Origin") %></th>
       <th class="col-md-2"><%= sort :reported_at, :as => _('Reported at') %></th>
-      <th class="col-md-1"><%= _('Actions') %></th>
```
Comment (stejskalleos) [inline app/views/fact_values/index.html.erb:20]: Should we remove it, or keep it, even when the list is empty?
What if some plugins are extending this column? An unlikely scenario in this case, but you never know

File: app/views/fact_values/_fact.html.erb [hunk 2]
```diff
@@ -25,15 +25,5 @@
...
-        <span id="view-fact-chart-<%= fact.id %>"></span>
-        <%= react_component('FactChart', {
-            path: fact_path(fact.fact_name_id),
-            title: fact.fact_name.name,
-            id: fact.id,
-            search: '/hosts?search=facts.' + fact.fact_name.name + '~~VAL1~'
-        }) %>
-      <% end %>
-  </td>
+
```
Comment (stejskalleos) [inline app/views/fact_values/_fact.html.erb]: Nit: remove trailing line

**Review by stejskalleos — CHANGES_REQUESTED** [review]
two things plus update the commit message with a description like in the PR.
Having a link there to the community would be nice too.

File: app/views/fact_values/index.html.erb (hunk 1 above)
Comment (girijaasoni) [reply app/views/fact_values/index.html.erb:20]: Only the view chart button is in the actions column code wise, IMHO we should remove it
@Ron-Lavi what do you think

Comment (Ron-Lavi) [reply app/views/fact_values/index.html.erb:20]: imho we can remove it as it will just stay empty and there's not seem to be any extensions to it.

**Review by stejskalleos — APPROVED** [review]
:green_apple: **LGTM**, works as expected

Thanks @girijaasoni !

**Pull request merged**